### DISCUSSION TITLE
Anyway to add files to "Output Files" view?

### DISCUSSION THREAD

**Discussion by longcasey** [github0null/eide#269] [Q&A]
I have a PostBuild Task that does the following:

`arm-none-eabi-objdump -d ${OutDir}/${ProjectName}.elf > ${OutDir}/${ProjectName}.objdump`

This works, no problems. However, the `*.objdump` file won't show in the "Output Files" view. i.e., it is in the filesystem at `${OutDir}` but not showing in the graphical view "Output Files". 

If I change output extension to be, say `*.objdump.map` then it will show up in the "Output  Files" graphical view.

Is there any way to change the filter so this *.objdump file can be seen without converting it to one of the predefined filters, which seems to be {`*.bin, *.elf, *.hex, *.map, *.map.view`} files?

awesome project btw. very neat lightweight IDE that just works.

on an unrelated note--does anyone know of any good syntax highlighters that can highlight objdump output?

Thanks for any help!

**Comment by github0null** [comment]
> Is there any way to change the filter so this *.objdump file can be seen without converting it to one of the predefined filters, which seems to be {*.bin, *.elf, *.hex, *.map, *.map.view} files?

We have a predefined filters: `/\.(?:elf|axf|out|sm8|a|lib|hex|ihx|bin|s19|s37|sct|icf|ld[s]?|map|map\.view|lst|lnp|params)$/i`

> arm-none-eabi-objdump -d ${OutDir}/${ProjectName}.elf > ${OutDir}/${ProjectName}.objdump

If you want to show disasm for full elf file, you can use right-click menu command

![image](https://github.com/github0null/eide/assets/43022536/0738e067-91da-4271-b41f-6c9abfb4efbd)

and then a ${ProjectName}.elf.edasm file will be generated and with highlight objdump output

![image](https://github.com/github0null/eide/assets/43022536/ece25bc3-8ce1-493b-b19e-31255bd2e7ed)
